plans/podman_build_test.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -eox pipefail
+
+rpm -q golang
+
+if [ -f /etc/fedora-release ]; then
+    export TMPDIR=/var/tmp
+fi
+
+git clone https://github.com/containers/podman
+
+cd podman
+dnf -y builddep rpm/podman.spec
```

Comment:
shouldn't this stuff be installed in the prepare step?

Reply:
That would be doable if we're only running podman tests. If we also want to do buildah, skopeo and anything else, then probably best done in individual tests.